packages/xod-client/src/core/styles/components/ReactDatasheet.scss

```
@@ -0,0 +1,89 @@
+span.data-grid-container, span.data-grid-container:focus {
+  outline: none;
+}
+
+.data-grid-container .data-grid {
+  table-layout: fixed;
+  border-collapse: collapse;
+}
+
+.data-grid-container .data-grid .cell.updated {
+    background-color: rgba(0, 145, 253, 0.16);
+    transition : background-color 0ms ease ;
+}
+.data-grid-container .data-grid .cell {
+  box-sizing: border-box;
+  user-select: none;
+  cursor: cell;
+  background-color: $input-bg;
+  color: $input-color-text;
+  transition : background-color 500ms ease;
```

```suggestion
   transition : background-color 100ms ease;
```

500ms is more than human reaction time, so looks boring. Animations look snappier when done in 50-200ms